pwiz_tools/Skyline/Model/Lib/BiblioSpecLite.cs

```diff
@@ -931,6 +931,18 @@ private byte[] CreateCache(ILoadMonitor loader, IProgressStatus status, int perc
             return cacheBytes;
         }
 
+        private SQLiteDataReader ExecuteReader(SQLiteCommand command)
+        {
+            try
+            {
+                return command.ExecuteReader();
+            }
+            catch (SQLiteException ex)
+            {
+                throw new IOException(string.Format(Resources.BiblioSpecLiteLibrary_CreateCache_Failed_reading_library_header_for__0__, FilePath), ex);

```

Comment:
Could we add a file existence check so that we can more clearly report the file is missing? I think that could be a more helpful message to the user than that we failed to read a header, whatever that means to a non-programmer.